packages/gitbook/src/components/Search/SearchModal.tsx

```diff
@@ -182,6 +183,14 @@ function SearchModalBody(
         } else if (event.key === 'Enter') {
             event.preventDefault();
             resultsRef.current?.select();
+
+            const queryIsQuestion = isQuestion(state.query);
+
+            onChangeQuery({
```

Comment:
- We should highlight what action we’re about to perform when pressing Enter, right now I don’t get any feedback on what I’m about to do. We can easily do this by moving the cursor such that either the first result (Ask) or the second result (first search result) is highlighted. I can then move up/down myself to deviate from the default.
- I think we need to always show the Ask AI prompt, right now we still conditionally show it. Even if we don’t think it’s a question, users should have the ability to ask it I think.